contracts/multisig-prover/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "multisig-prover"
-version = "0.6.0"
```

Comment:
shouldn't have to manually change these :/

Reply:
we needed to for the manual release, because the release action can only work from the HEAD of main, but here this should be indeed reverted